src/tools/kata-ctl/src/check.rs

```diff
@@ -6,32 +6,22 @@
 // Contains checks that are not architecture-specific
 
 use anyhow::{anyhow, Result};
-// See: https://github.com/kata-containers/kata-containers/issues/5438
-#[cfg(any(
-    target_arch = "aarch64",
-    target_arch = "powerpc64le",
-    target_arch = "x86_64"
-))]
 use reqwest::header::{CONTENT_TYPE, USER_AGENT};
 use serde_json::Value;
-#[cfg(any(
-    target_arch = "aarch64",
-    target_arch = "powerpc64le",
-    target_arch = "x86_64"
-))]
 use std::collections::HashMap;
-use std::fs;
 
 const KATA_GITHUB_URL: &str =
     "https://api.github.com/repos/kata-containers/kata-containers/releases/latest";
 
+#[cfg(test)]
```

Comment:
@hbrueckner this is not a module test, so I think you should remove the #cfg(test) annotation.

Reply:
You are right, the static checks CI job failed due to this... I will push an update later today.